CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.8.0 (2020-02-18)
+
+- Added the `strict` property to the datepicker input directive to indicate whether the format of the date value must match the format from the `dateFormat` value.  [#114](https://github.com/blackbaud/skyux-datetime/pull/114)
```

Comment:
Should `value` at the end be `property`? Or is it accurate either way? (If it should be `property`, we need to update the pending docs.)

Reply:
I think `value` works, because it's referring to the value that the user provides.

Reply:
Good deal.